client/blocks/jitm/templates/modal-style.scss

```diff
@@ -208,8 +212,7 @@ $wpcom-modal-footer-height: 30px + ( $wpcom-modal-footer-padding-v * 2 );
 	@media ( min-width: $wpcom-modal-breakpoint ) {
 		display: block;
 		background-color: var( --studio-blue-5 );
-		height: 100%;
-		width: 90%;
+		min-width: 45%;
```

Comment:
At smaller browser width, the image renders quite small, because of the `min-width` set in the text area. Changing the approach so that we set the `min-width` for the image to ensure the image renders in a reasonable size.